### ISSUE TITLE
bar component: add focus to the bar element

### ISSUE THREAD

**Issue by Lokanathan-k** [SAP/fundamental-styles#2593]
add focus to the bar element, focus is not as per the fiori spec

**Describe the bug**

<img width="345" alt="Screenshot 2021-07-20 at 6 45 21 PM" src="https://user-images.githubusercontent.com/53534379/126330592-b6f45c21-28ba-4276-b7e3-4f4d8fbfcceb.png">

<img width="327" alt="focus error " src="https://user-images.githubusercontent.com/53534379/126330168-0ce7f0ad-c3e2-46d7-a270-24e95ceaaecf.png">

**Comment by droshev** [comment]
@Lokanathan-k first of all, how is this related to the bar? Bar is just container component.

**Comment by Lokanathan-k** [comment]
> how is this related to the bar? Bar is just container component


@droshev 
in the popover header and few other components use the fd-bar as the header container these container doesnot have a focus the the element as per fiori spec.

**Comment by Lokanathan-k** [comment]
not an issue
